app/models/user.rb

```diff
@@ -42,10 +42,8 @@ def send_invitation(user_id)
   end
 
   def confirm_invites(user)
-    send_invitation.update_attributes(confirmed: true)
-    Friendship.create!(friend_id: send_invitation.user_id,
-                  user_id: send_invitation.friend_id,
-                  confirmed: true)
+    friendship = inverse_friendships.where(user_id: user).first
+    friendship.status = true
```

Comment:
Please address this. The reason why it's a bit problematic is you are trying to confirm the friendship in the user model when the logic should be happening on the friendship itself